packages/commands/src/components/command-menu.js

```diff
@@ -199,6 +199,9 @@ export function CommandMenu() {
 	useShortcut(
 		'core/commands',
 		( event ) => {
+			// Bails to avoid obscuring the effect of the preceding handler(s).
+			if ( event.isDefaultPrevented() ) return;
```

Comment:
I notice elsewhere in the repo, the check is usually for the property `event.defaultPrevented` rather than calling `isDefaultPrevented()` that gets exposed via React's synthetic event. Shall we stick with `event.defaultPrevented` for consistency with the other checks?

Reply:
I don’t know. I would think elsewhere the check are of plain DOM events. Here, `event` is a `SyntheticEvent` so wouldn't it have to be `event.nativeEvent.defaultPrevented`?

Reply:
Something that might have to do with the variations we see in the repo is we have both `useShortcut` and `useKeyboardShortcut`. The former utilizes react’s synthetic events and the latter doesn't.

Reply:
Okay, elsewhere in the repo it's **always** `defaultPrevented` (and indeed 🤦 there’s direct access to it—no need for `nativeEvent`). So a7a5d7c. Thanks for pointing this out 🙇